src/game/client/neo/ui/neo_hud_startup_sequence.cpp

```diff
@@ -87,7 +87,7 @@ struct FlavourTextEntry
 
 void CNEOHud_StartupSequence::DrawNeoHudElement()
 {
-	if (!ShouldDraw())
+	if (!ShouldDraw() || IsLocalPlayerSpectator())
```

Comment:
```suggestion
	if (!ShouldDraw() || IsLocalPlayerSpectator() || engine->IsDemo())
```
Or whatever it was. Might be a useful addition